core/model/modx/processors/element/propertyset/updatefromelement.class.php

```diff
@@ -9,44 +10,39 @@
 
 class modPropertySetUpdateFromElementProcessor extends modPropertySetUpdateProcessor {
     public $languageTopics = array('propertyset', 'category', 'element');
-
-    /** @var modPropertySet|null */
-    public $object;
```

Comment:
It's not about setting the variable, but about it's comment, where you declare it's type - so you can have a proper autocompletion in IDE.